tests/test_logging_config.py

```diff
@@ -251,6 +251,14 @@ def test_1_9_config(self):
         finally:
             conf.remove_option('core', 'task_log_reader', remove_default=False)
 
+    def test_loading_remote_logging_with_wasb_handler(self):
+        """Test if logging can be configured successfully for Azure Blob Storage"""
+        from airflow.logging_config import configure_logging
+        conf.set('core', 'remote_logging', 'True')
+        conf.set('core', 'remote_log_conn_id', 'some_wasb')
+        conf.set('core', 'remote_base_log_folder', 'wasb://some-folder')
+        configure_logging()
```

Comment:
I know we were just testing that it loaded without error, but it always feels a bit odd to not see any assertions in a test.

How about we add something like

```python
import airflow.utils.log.wasb_task_handler.WasbTaskHandler
logger = logging.getLogger('airflow.task.x')
self.assertIsInstance(logger.handlers[0], airflow.utils.log.wasb_task_handler.WasbTaskHandler)
```

Reply:
(I've done that from memory- that may not be the property name on logger)

Reply:
Sure. I'll add that.

Reply:
@ttanay Any chance you'd have a chance to look at this last change?

Reply:
Hey @ashb 
Didn't get a chance to look at it. Exams :/
Will get it done by Tuesday.

Reply:
Hey @ashb 
I tired this after `configure_logging` is called in the change above, and also in a DAG:
```
import logging
logger = logging.getLogger('airflow.task.x')
```
But, the `logger.handlers` was an empty list `[]`.

Am I missing something?

Reply:
Hmmm, I'll double check. Did you also try with just `logging.getLogger('airflow.task')`

Reply:
No, but, it works now. Thanks!

Reply:
Ah that makes sense. `airflow.task.x` isn't directly configured with handlers (hence the empty list) but when emitting messages the logging framework will go up the hierarchy until it finds some (that's a little bit simplified, but close)

Reply:
Oh. Now I get it. Thanks!